.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ jobs:
             if [ $? -eq 0 ]; then
               chmod +x ~/codecov
               cd ~/repo
-              ~/codecov -t ${CODECOV_TOKEN}
+              ~/codecov -t ${CODECOV_TOKEN} -v
```

Comment:
I assume codecov is sanitizing this output?

Reply:
yeah the `-v` just prints the length of the token, not the token itself, was using it to try to debug the "not showing any relative coverage changes" issue a while ago and it _seemed_ to be more better with the flag on but thats probably irrelevant